### DISCUSSION TITLE
Loss of Experience Builder text formatting

### DISCUSSION THREAD

**Discussion by ccuevas03** [EsriPS/arcgis-assistant-feedback#66] [Q&A]
Has anyone else had the issue, where, if you update the json in ArcGIS Assistant, and you go back to the original item, in this case, Experience Builder, and all the text formatting is messed up?  When viewing it, it still shows correctly, but when editing, it looks like a mess.

Also, when you update the json in 'Data', you are supposed to update the config.json in 'Resources' as well, right?

Utilizing ArcGIS Enterprise Portal for ArcGIS 10.9.1 and Experience Builder 1.5.

**Comment by SasquatchPhil** [comment]
seeing issues where editing the config.json for an experience builder project results in completely removing everything after saving...   ??? ArcGIS Online,

**Reply by ccuevas03** [reply]
So, you can't see your experience builder project at all anymore in AGOL?  Mine altered all the formatting (in Portal for ArcGIS).  It will still display correctly as I originally formatted it when viewing, but when editing the formatting is all wonky.... I'm thinking ArcGIS Assistant is not very friendly with Experience Builder... :(  I don't wan't to lose my Experience app altogether.... that's very concerning!

**Reply by jpeterson** [reply]
@SasquatchPhil and @ccuevas03 thanks for bringing this up. I'm looking into what could be happening here. If you can share any more details about the edits you're seeing, that would be very helpful. One question off the bat – is this a configured Experience Builder app, or a customized Experience Builder **Developer Edition** app?

**Comment by SasquatchPhil** [comment]
yep, after editing a small part in the config.js and saved, refreshed the browser and it was missing. something is not processing correctly there...  refreshed the exb template and its erroring because nothing is there withing the config.js - crazy.  good thing it was a working copy.   yes be careful there.

**Comment by ccuevas03** [comment]
Hi Josh,
Mine was configured Experience Builder app in Portal for ArcGIS 10.9.1.  I was just trying to change the page numbers in the json.  I know this feature will be available in 11, but we will not be updating for a few more months.  Also, eventually the formatting did carry over to the live view of the app, and I had to go through and fix all the formatting.  Formatting that was lost was in text boxes, and within icons and button icons.  For example, if I had the 'Home' icon selected, it no longer saw that it was pointing to that icon.  For button icons, it did the same on some, and also removed some of the formatting for the icon for 'Selected' and 'Hover' settings.
